### DISCUSSION TITLE
would you add option for yearly planning page with this Year block?

### DISCUSSION THREAD

**Discussion by www2341** [YU000jp/logseq-plugin-show-weekday-and-week-number#82] [Show and tell]
hi, I'm newbie of logseq. 

I'm looking for planing plugin, like yours.
thank you, it's good for me.

weekly page added with This Week block. 
![image](https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/assets/17059492/9a3508fe-83ad-46f9-98a5-0848daa7c46c)

how about add This Year block for year page too?
not only page tagged, create This Year block and put [[2023-W01]] ~~ [[2023-W52]] link like THIS WEEK block.
![image](https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/assets/17059492/ed7a5884-b8a6-45fe-8110-5f0de75231f7)

it would be very useful for me to planing yearly.

thank you so much.

**Comment by YU000jp** [comment]
Firstly, there is a reason why I chose Page-tags over links. It's to prevent any degradation in the performance of Logseq application. Having an abundance of links can strain the database without yielding significant benefits.

Furthermore, even if we were to create week-number links on pages for that year, the sheer volume of links alone doesn't seem likely to contribute significantly to planning. When it comes to aggregating reviews or engaging in year-long planning, I believe we should explore alternative approaches.
I'm also exploring.😅

**Reply by www2341** [reply]
um... i think back-link is good for planning and I've never feel slow.

i'm planning complexly, one is project goal base, one is weekly schedule base like below:

2023 (page)
  - project1
    - goal1 [[2023-W23]]
    - goal2 [[2023-W24]]
  - project2 
    - goal7 [[2023-W23]]
    - goal8 [[2023-W26]]
      
#### this year  
  - [[2023-W23]]
    - other plan1. ~~~
    - other plan2. ~~~ 

and then, goto [[2023-W23]] page then

there are goals for this week in back-link.
project1. goal1
project2. goal7
and
weekly plans

with these golas, I'm planning for day-by-day.

in that case, I've never feel slow or some performance issues.

if my suggest is not fit concept, or some issues, 
how about option or shortcut to just generate [[2023-W01]] - [[2023-W52]] on current page?

I'm very thank you for this plugin.

**Reply by YU000jp** [reply]
Add as slash command. https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/pull/88


https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/wiki/Slash-Command

**Reply by www2341** [reply]
thank you, it's very good for me.
